### PR TITLE
Added essentials stuffs and more detailed idle

### PR DESCRIPTION
**Pull request by NateKomodo**
will add latest release upon merge

### CODE REVIEW

**Comment by thebeanogamer** [comment]
image

File: CyberDisc-RichPresence/CDRichPresence.cs [hunk 1]
```diff
@@ -1,228 +1,247 @@
...
+            {
+                DoEliteUpdates(url);
+                return;
+            }
+            SetRP("Idle", "", "cd");
+        }
+
+        private void DoEliteUpdates(string url)
+        {
+            SetRP("Working on elite", "Wait what this isnt possible", "game");
```
Comment (thebeanogamer) [inline CyberDisc-RichPresence/CDRichPresence.cs:110]: Wot?

**Comment by NateKomodo** [comment]
blame VS

File: CyberDisc-RichPresence/CDRichPresence.cs (hunk 1 above)
Comment (NateKomodo) [reply CyberDisc-RichPresence/CDRichPresence.cs:110]: meme

**Comment by thebeanogamer** [comment]
> 
> 
> blame VS

https://www.jetbrains.com/rider/

File: CyberDisc-RichPresence/CDRichPresence.cs (hunk 1 above)
Comment (thebeanogamer) [inline CyberDisc-RichPresence/CDRichPresence.cs:110]: Remove to merge. It's C# so it's already bloated. We don't need more of that.

**Review by thebeanogamer — CHANGES_REQUESTED** [review]

File: CyberDisc-RichPresence/CDRichPresence.cs (hunk 1 above)
Comment (NateKomodo) [reply CyberDisc-RichPresence/CDRichPresence.cs:110]: its already in the deployed version, and ima have to say one extra method isnt going to do shit

File: README.md [hunk 2]
```diff
@@ -1,11 +1,2 @@
-# Cyber Discovery Rich Presence
```
Comment (thebeanogamer) [inline README.md:1]: Have you actually checked the diff?

File: CyberDisc-RichPresence/obj/Debug/CyberDisc-RichPresence.csproj.FileListAbsolute.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+C:\Users\natko\Source\Repos\cd-rich-presence\CyberDisc-RichPresence\obj\Debug\CyberDisc-RichPresence.csprojAssemblyReference.cache
```
Comment (thebeanogamer) [inline CyberDisc-RichPresence/obj/Debug/CyberDisc-RichPresence.csproj.FileListAbsolute.txt:1]: This all makes reference to local file locations on your computer.

File: CyberDisc-RichPresence/CDRichPresence.cs (hunk 1 above)
Comment (thebeanogamer) [reply CyberDisc-RichPresence/CDRichPresence.cs:110]: Fun fact: Not your choice

**Review by thebeanogamer — CHANGES_REQUESTED** [review]

File: CyberDisc-RichPresence/CDRichPresence.cs (hunk 1 above)
Comment (NateKomodo) [reply CyberDisc-RichPresence/CDRichPresence.cs:110]: i mean its my project....

File: README.md (hunk 2 above)
Comment (NateKomodo) [reply README.md:1]: i literally just downloaded it and made changes to the cs file

File: CyberDisc-RichPresence/obj/Debug/CyberDisc-RichPresence.csproj.FileListAbsolute.txt (hunk 3 above)
Comment (NateKomodo) [reply CyberDisc-RichPresence/obj/Debug/CyberDisc-RichPresence.csproj.FileListAbsolute.txt:1]: As long as you use the release build or build it with the DLLs within the project folder, it works

**Comment by NateKomodo** [comment]
you know what im just going to upload the cs file

**Pull request closed without merging**